upup/pkg/fi/cloudup/awsup/aws_utils.go

```diff
@@ -221,6 +221,15 @@ func ELBv2Tags(tags map[string]string) []*elbv2.Tag {
 	return elbv2Tags
 }
 
+// EventBridgeRulePrefix returns a suitable prefix for event bridge rules given their length restrictions
+func EventBridgeRulePrefix(cluster string) string {
```

Comment:
I would call this more like ClusterName40. But it's just a suggestion. Seems more consistent with GetResourceName32 that we use in other places.